invenio_circulation/config.py

```diff
@@ -32,6 +32,9 @@
 CIRCULATION_DOCUMENT_RETRIEVER_FROM_ITEM = None
 """Function that returns the Document PID of a given Item PID."""
 
+CIRCULATION_STATES_LOAN_REQUEST = ['PENDING']
```

Comment:
Introduced variables for the list of states when loan is considered PENDING and CANCELLED so that the system is more configurable

Reply:
👍